BookingsApi/BookingsApi.UnitTests/DAL/Services/FeatureFlagServiceTests.cs

```diff
@@ -50,9 +53,10 @@ public void GetFeatureFlag_Should_Return_False_for_StaffMember_Feature()
         [Test]
         public void GetFeatureFlag_Should_Return_True_for_EJud_Feature()
         {
+            _mocker.Mock<IFeatureToggles>().Setup(opt => opt.EJudFeature()).Returns(true);
             _mocker.Mock<IOptions<FeatureFlagConfiguration>>().Setup(opt => opt.Value).Returns(new FeatureFlagConfiguration()
             {
-                EJudFeature = true
+                StaffMemberFeature = false
             });
```

Comment:
Why's this test been changed to staffmember feature: false? when the name of the test is should return true for Ejud Feature?

Reply:
just added it in case would fail to initialise the service

Reply:
removed